### PR TITLE
Gamma and workspace documentation updated

### PR DESCRIPTION
**Pull request by mdacoca**
### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into the [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: site/content/en/docs/manual/basics/workspace.md [hunk 1]
```diff
@@ -1,38 +1,122 @@
...
+   transparency of the grid overlay.
+
+### Changing color settings of image is CVAT
+
+To change the color setting of the image is CVAT, do the following:
+
+1. Open the **Image Settings** panel.
+2. Use the slider to change the color quality.
+
+There are four color quality settings in CVAT:
```
Comment (klakhov) [inline site/content/en/docs/manual/basics/workspace.md:68]: Please, add a note that these settings affect only how users see image, not the image itself

File: site/content/en/docs/manual/basics/workspace.md [hunk 2]
```diff
@@ -1,38 +1,122 @@
...
+2. Locate and check the box that
+   allows you to overlay a grid on the image.
+3. Specify the grid cell size in square millimeters
+   by entering the desired number in the **Size** field.
+4. From the **Color** drop-down list,
+   select the color of the grid.
+5. Use the Opacity slider to change the
+   transparency of the grid overlay.
+
+### Changing color settings of image is CVAT
```
Comment (klakhov) [inline site/content/en/docs/manual/basics/workspace.md:61]:
```suggestion
### Changing color settings of image in CVAT
```

File: site/content/en/docs/manual/basics/workspace.md (hunk 1 above)
Comment (klakhov) [reply site/content/en/docs/manual/basics/workspace.md:68]: Maybe we should put that note in `Image settings` section

**Review by klakhov — APPROVED** [review]

File: site/content/en/docs/manual/basics/workspace.md [hunk 3]
```diff
@@ -1,38 +1,126 @@
...
+![Image Brightness](/images/image-settings-contrast.png)
+
+**Saturation** describes the intensity of the color.
+
+![Image Saturation](/images/image-settings-saturation.png)
+
+**Gamma** correction can be used to
+control the overall brightness of an image
+
+![Image Saturation](/images/image-settings-gamma.jpg)
```
Comment (nmanovic) [inline site/content/en/docs/manual/basics/workspace.md:91]:
```suggestion
![Gamma Correction](/images/image-settings-gamma.jpg)
```

**Pull request merged**